environment.yml

```diff
@@ -1,19 +1,19 @@
 name: refred
 channels:
-  - mantid/label/main
+  - mantid/label/nightly
   - conda-forge
   - defaults
 dependencies:
-  - python=3.8
-  - mantid=6.7.0
+  - python=3.10
+  - mantid>6.9.20240318
   - qt>=5.12.9,<6
```

Comment:
have you checked [mantid-developer-linux.yml](https://github.com/mantidproject/mantid/blob/main/mantid-developer-linux.yml) to verify the qt, pyqt, and qtpy dependencies agree?